build_docs.pl

```diff
@@ -337,9 +337,13 @@ sub check_links {
         say $link_checker->report;
     }
     else {
+      if ( $Opts->{warnlinkcheck} ){
```

Comment:
Separately, I wonder if it would be good to pass this in as a boolean variable (to say whether broken links should be treated as "warn" or "die"), and then call the function as `check_links($build_dir, $Opts->{warnlinkcheck})` so we could potentially call this function in different ways in different scenarios without needing to worry about the command line args.

Reply:
I'm not sure I understand how this helps us avoid command line args. Wouldn't you still need to specify "--warnlinkcheck=xyz"?

Reply:
Sorry, it wouldn't affect the command line args at all. It just makes the `check_links` function easier to use in the future (if you want the warn-vs-die decision to be based on something other than the argument). It's probably pre-mature optimization at this point, though.